packages/node_modules/@webex/redux-module-conversation/src/__snapshots__/actions.test.js.snap

```diff
@@ -509,20 +509,7 @@ Array [
 ]
 `;
 
-exports[`redux-module-conversation actions #submitAdaptiveCardActions can start request when card action submitted 1`] = `
-Array [
-  Object {
-    "payload": Object {
-      "activity": Object {
-        "activity": Object {
-          "id": "7erfber34",
-        },
-      },
-    },
-    "type": "adaptiveCards/ACKNOWLEDGE_ADAPTIVE_CARD_SUBMIT_ACTION",
-  },
-]
-`;
+exports[`redux-module-conversation actions #submitAdaptiveCardActions can start request when card action submitted 1`] = `Array []`;
```

Comment:
The change in the action test caused this test snapshot to empty and not actually do anything.